### ISSUE TITLE
Release as python library

### ISSUE THREAD

**Issue by georg-schwarz** [binance/binance-public-data#149]
Hey! Unfortunately, I didn't find a template on how you want feature requests to be formatted, so I just wrote a user story. Hope that is okay :)

## Story
As a data scientist or application developer, I'd like to use the python code as a library instead of copying code around in order to incorporate the scripts into my usual development workflow.

**Comment by 2pd** [comment]
> Are you having any luck downloading historical data? Doesn't seem to be working for me.

Do you see any issue  or any error?

**Comment by 2pd** [comment]
> Hey! Unfortunately, I didn't find a template on how you want feature requests to be formatted, so I just wrote a user story. Hope that is okay :)
> ## Story
> 
> As a data scientist or application developer, I'd like to use the python code as a library instead of copying code around in order to incorporate the scripts into my usual development workflow.

Yeah, that's in the plan.  If you can share your requirements in detail, it will be helpful for us to make it better.

thanks.

**Comment by georg-schwarz** [comment]
I think as the first step publishing as a versioned library would be a great addition, e.g. on [pypi](https://pypi.org/).
That would allow me (as a user of a lib) to install and update it via pip cli (e.g. `pip3 install binance-client`) instead of copying your code into my repository.

Hope that helps?
